### PR TITLE
Bump timeout

### PR DESCRIPTION
**Pull request by eawooten**

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const https = require('https');
 
 class HTTP_Client {
 
-    constructor(timeout = 12000){
+    constructor(timeout = 60000){
```
Comment (NicLee0521) [inline lib/http.js:6]: would it make sense to make the timeout equivalent to the worker timeout or is that just asking for trouble?

Comment (eawooten) [reply lib/http.js:6]: I would be concerned about leaving a request open that long, and I want us to get rid of this entire HTTP client sooner than later anyway.

Comment (NicLee0521) [reply lib/http.js:6]: very fair

**Review by NicLee0521 — APPROVED** [review]

**Review by chrisguitarguy — COMMENTED** [review]
Imo a better approach here would be to allow the timeout to be passed "up" through the chain of constructors. Eg. https://github.com/AgencyPMG/pmg-criteo-api/blob/004f13e858d11b0c6dbac8f42ebab27638dcc729/lib/api_client.js#L6

Then the timeout can bet tweaked without upstream code changes.

Should take a timeout.

Or if you want a more aggressive refactoring, there's no real reason to extend the HTTP Client class at all. Inject it into the constructor as a collaborator, then you can configure it however you want on creation.

I'd also so that stuff like `apiGet` get in `API_Client` belong as methods in the http client itself (API client should not exist, imo). Along with host/port, protocol, etc (all that could be encapsulating into a Base URL, tbh).

Depends on how much work you want to do here.

**Review by deepak0510 — APPROVED** [review]

**Comment by eawooten** [comment]
> Imo a better approach here would be to allow the timeout to be passed "up" through the chain of constructors.

@chrisguitarguy Yeah I am working on the "better" fix concurrently, but wanted to have this incase they needed it out in a hurry. I will implement this feedback in the long term solution.

**Review by NicLee0521 — APPROVED** [review]

**Review by chrisguitarguy — APPROVED** [review]

**Pull request merged**